app/serializers/search/article_serializer.rb

```diff
@@ -0,0 +1,39 @@
+module Search
+  class ArticleSerializer
+    include FastJsonapi::ObjectSerializer
+
+    attributes :id, :approved, :body_text, :class_name, :cloudinary_video_url,
+               :comments_count, :experience_level_rating, :experience_level_rating_distribution,
+               :featured, :featured_number, :flare_tag, :hotness_score, :language,
+               :main_image, :path, :positive_reactions_count, :published,
+               :published_at, :reactions_count, :reading_time, :score, :title,
+               :video_duration_string
+
+    # video_duration_in_minutes in Elasticsearch is mapped as an integer
+    # however, it really is a string in the format 00:00 which is why we
+    # added an extra field to handle that string
+    attribute :video_duration_string, &:video_duration_in_minutes
+    attribute :video_duration_in_minutes, &:video_duration_in_minutes_integer
+
+    attribute :tags do |article|
+      article.tags.map do |tag|
+        { name: tag.name, keywords_for_search: tag.keywords_for_search }
```

Comment:
`keywords_for_search` is populated by admins, correct?

Reply:
Looks like it! I honestly had no idea we did this but after looking at some prod data I kinda love it. Its like our own little built in synonyms generator